src/cmdlib.sh

```diff
@@ -225,17 +225,18 @@ prepare_build() {
     export fastbuilddir
 
     # Allocate temporary space for this build
-    tmp_builddir=${workdir}/tmp/build
+    tmp_builddir="${workdir}"/tmp/"build${IMAGE_TYPE:+.$IMAGE_TYPE}"
```

Comment:
Can the quotes get simplifed to:

```suggestion
    tmp_builddir="${workdir}/tmp/build${IMAGE_TYPE:+.$IMAGE_TYPE}"
```